alpine/alpine-unfixed/alpine.go

```diff
@@ -0,0 +1,202 @@
+package alpineunfix
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/parnurzeal/gorequest"
+
+	"github.com/PuerkitoBio/goquery"
+	"github.com/spf13/afero"
+	"golang.org/x/xerrors"
+
+	"github.com/aquasecurity/vuln-list-update/utils"
+)
+
+const (
+	alpineDir = "alpine-unfixed"
+	baseUrl   = "https://security.alpinelinux.org"
+	retry     = 3
+)
+
+type Updater struct {
+	vulnListDir string
+	appFs       afero.Fs
+	baseURL     string
+	retry       int
+	CVEUrl      string
+}
+
+func NewUpdater() *Updater {
+	updater := &Updater{
+		vulnListDir: utils.VulnListDir(),
+		appFs:       afero.NewOsFs(),
+		baseURL:     baseUrl,
+		retry:       retry,
+	}
+	return updater
+}
+func (u Updater) Update() (err error) {
+	baseData := make(map[string]string)
+	dir := filepath.Join(u.vulnListDir, alpineDir)
+	log.Printf("Remove Alpine directory %s", dir)
+	if err := u.appFs.RemoveAll(dir); err != nil {
+		return xerrors.Errorf("failed to remove Alpine directory: %w", err)
+	}
+	if err := u.appFs.MkdirAll(dir, 0755); err != nil {
+		return xerrors.Errorf("Failed creating dir %s: %w", dir, err)
+	}
+
+	log.Println("Fetching branch data...")
+	activeBranches, err := getActiveReleases(u.baseURL)
+	if err != nil {
+		return xerrors.Errorf("Failed getting active branches: %w", err)
+	}
+	activeBranches = append(activeBranches, "3.10-main")
```

Comment:
Let's start with hard-coded versions. we may be able to contribute to their source. I think `show_index` should support JSON as well.
https://gitlab.alpinelinux.org/kaniini/secfixes-tracker/-/blob/master/secfixes_tracker/views.py#L9-12

Reply:
no it does not support JSON
https://gitlab.alpinelinux.org/kaniini/secfixes-tracker/-/blob/master/secfixes_tracker/views.py#L9-12
has only 
`@accept('text/html')`

Reply:
I know it is not supported now. I meant we can contribute to secfixes_tracker so that it can support JSON API in `show_index`.